README.md

```diff
@@ -1,118 +1,173 @@
+```
+1 Natural Language Classifier free per month.
+1000 API calls free per month
+4 Training Events free per month
+```
 
-#### Create a Natural Language Classifier instance on IBM Cloud
+Once the service is created the `Credentials` will be on the page. Click `Show` to make them visible and copy them for later use when you [Configure credentials](#4-configure-credentials). You can always get to the credentials by clicking the `Service credentials` on the left.
 
-* Go to IBM Cloud catalog and select [Watson Natural Language Classifier](https://console.bluemix.net/catalog/services/natural-language-classifier)
+### 3. Train the NLC model
 
-#### Train the Natural Language classifier using training data
+* In your project, under the `Assets` tab and `Models` click `+ New Natural Language Classifier model` to bring up the `New Classifier` GUI page:
 
-Training a classifier is easy and can be done in 2 ways. The first way is more technical in nature and involves directly calling the API. The second method leverages the GUI based tooling.
+![](https://github.com/IBM/pattern-images/tree/master/watson-studio/NewNLCclassifierGUI.png)
 
-#### First Method
-Simply, provide training data in a Watson NLC compatible format (.csv) and POST a request to the Watson NLC `/classifiers` REST endpoint.
+* Add the data to your project by clicking the `Browse` button in the right-hand `Upload to project` section and browsing to this repo. Choose both `data/Email-trainingdata- 20k.csv` and `Email-testingdata.json`.
 
-* Open`data/Email-trainingdata-20k.csv` to view the data format
-* Train Watson NLC
+* Drag and drop the `Email-trainingdata- 20k.csv` file you uploaded to the `Create a Class` box:
 
-    ```
-  curl -X POST -u username:password -F training_data=@Email-trainingdata-20k.csv \
-   -F training_metadata="{\"language\":\"en\",\"name\":\"My Classifier\"}" \
-  "https://gateway.watsonplatform.net/natural-language-classifier/api/v1/classifiers"
-    ```
-#### Second Method 
+![video-to-gif](doc/source/videos/dragNdropDataNLC.gif)
 
-Access the GUI based tooling by logging into your [IBM Cloud Dashboard](https://console.us-east.bluemix.net/dashboard/services). 
+* Click the `Train model` button to begin training. The model will take around an hour to train.
 
-In the Services area, click your Natural Language Classifier service tile to open the instance dashboard. Once in the service dashboard, click `Open tool` and upload your training data.
+* To check the status of the model, and access it after it trains, go to your project in the `Assets` tab, `Models` section. The model will show up when it is ready. Double click to see the `Overview` tab.
 
```

Comment:
suggestion -  "`Assets` tab of the `Models` section."

Reply:
done